inc/namespace.php

```diff
@@ -47,6 +40,9 @@ function bootstrap( Module $module ) {
 
 	// Read config/robots.txt file into robots.txt route handled by WP.
 	add_filter( 'robots_txt', __NAMESPACE__ . '\\robots_txt', 10 );
+
+	// Add sitemap to robots.txt.
+	add_filter( 'robots_txt', __NAMESPACE__ . '\\add_sitemap_index_to_robots', 11, 2 );
```

Comment:
It did not seem to do this automatically. That might be because Yoast can tell I'm working locally and not on a production environment, but I couldn't figure out how to fool it into thinking I was on production and I figured better safe than sorry.

Reply:
Yeah, quick search of the Yoast codebase shows no actual use of the `robots_txt` filter 👍

Reply:
I guess because the robots meta tag is more modern? Maybe Yoast knows something we don't? `¯\_(ツ)_/¯`